console: Add comments about Kerberos configs (#779)
redpanda-data/documentation: docs/platform/console/reference/config.mdx

    # Valid mechanisms are: PLAIN, SCRAM-SHA-256, 
    # SCRAM-SHA-512, GSSAPI, OAUTHBEARER and AWS_MSK_IAM.
    mechanism: PLAIN

>     # Kerberos implements the GSSAPI protocol. Below configurations
>     # can be used to connect Console to a Kerberos protected Kafka API.

Reviewer: ```suggestion
    # Kerberos implements the GSSAPI protocol. Use the following configurations
    # to connect Redpanda Console to a Kerberos-protected Kafka API.
```
